virginia.clubrunning.org/index.html

```diff
@@ -107,15 +107,15 @@ <h3 class="pad-bottom">Meets. <span class="weak italic">Hoos gonna give it to ya
 		</div>
 	</div>
 
-	<div class="splash image" style="background-image: url('static/images/splash/landing/0073.jpg')"></div>
+	<div class="splash image" style="background-image: url('static/images/splash/landing/IMG_5802.jpg')"></div>
```

Comment:
I don't like the asymmetry in the new landing photo. I agree that the homepage photos need to be updated, but I think that should probably be its own PR. I request that you leave the landing photos alone for this PR